scripts/utils/trading_strategy_helpers.js

```diff
@@ -56,8 +58,18 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
    * @returns {boolean} whether owned is indeed owned only by master
    */
   const isOnlySafeOwner = async function (masterAddress, owned) {
-    const ownedSafe = typeof owned === "string" ? await getSafe(owned) : owned
-    const ownerAddresses = await ownedSafe.getOwners()
+    let ownedSafeInstance
```

Comment:
@andre-meyer These code changes here were only made for debugging, right? We don't need to adapt to it?

Reply:
Federico suggested to split this check into a different function. We do need the ability to fund/order with not-yet-created brackets though, for this to work with precalculated addresses.

Reply:
To my understanding, we don't need these changes to `isOnlySafeOwner` anymore since PR #441.